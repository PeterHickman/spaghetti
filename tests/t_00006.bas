10 REM Handle booleans
20 IF 1 = 1 AND 2 = 2 PRINT "Ok 1"
30 IF 1 = 2 OR 2 = 2 PRINT "Ok 2"
40 IF 1 = 1 OR 2 = 1 PRINT "Ok 3"