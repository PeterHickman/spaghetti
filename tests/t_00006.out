Ok 1
Ok 2
Ok 3
